resources/cdk/photo_asset_manager/SPECIFICATION.md

```diff
@@ -1,130 +1,128 @@
 # Photo Asset Management Tech Spec
 
-This is the Tech Spec for the Photo Asset Management cross services example. This doc details the specifics of the deliverable app.
+This is the tech spec for the Photo Asset Management cross-service example. This doc details the specifics of the deliverable app.
```

Comment:
Not sure if anyone will be totally clear what tech spec means here, as this is one of those weird words that's used super specifically in many companies and I don't want to confuse people.

I would a) either clarify (e.g. just append or prepend "A technical specification document outlines the necessities for a product, system, or project. "), or can just say b)
```suggestion
These are the technical specifications for the Photo Asset Management cross-service example. This doc details the specifics of the deliverable app.
```

Reply:
Please create a follow up PR.